src/modules/systemlib/mixer/mixer.h

```diff
@@ -518,18 +535,30 @@ enum class MultirotorGeometry : MultirotorGeometryUnderlyingType;
 	virtual unsigned		mix(float *outputs, unsigned space, uint16_t *status_reg);
 	virtual void			groups_required(uint32_t &groups);
 
+	/**
+	 * @brief      Update slew rate parameter. This tells the multicopter mixer
+	 *             the maximum allowed change of the output values per cycle.
+	 *
+	 * @param[in]  slew_rate_max  The maximum slew rate.
+	 */
+	virtual void		update_slew_rate(float slew_rate_max) {_slew_rate_max = slew_rate_max;}
```

Comment:
Since `_slew_rate_max` gets reset on every cycle, I think the name `update_slew_rate` is misleading. It should probably be called `set_slew_rate_for_next_cycle`, or `set_slew_rate_once`.